File: February 26, 2026.py
def count_letters_and_numbers(s):

    number = 0
    letter = 0

    for char in s:
        if char.isdigit():
            number += 1
        elif char.isalpha():
            letter += 1
    
    letter_word = "letter" if letter == 1 else "letters"
    number_word = "number" if number == 1 else "numbers"
    
    return f"The string has {letter} {letter_word} and {number} {number_word}."

File: February 26, 2026.js
function countLettersAndNumbers(str) {
  
  const numberRegex = /[0-9]/;
  const letterRegex = /[a-zA-Z]/;
  let number = 0;
  let letter = 0;

  for(let char of str) {
    if(numberRegex.test(char)) {
      number++;
    } else if(letterRegex.test(char)) {
      letter++;
    }
  }

  let letterWord = (letter === 1) ? "letter" : "letters";
  let numberWord = (number == 1) ? "number" : "numbers";

  return `The string has ${letter} ${letterWord} and ${number} ${numberWord}.`;
}